src/marqo/vespa/vespa_client.py

```diff
@@ -162,11 +163,14 @@ def wait_for_application_convergence(self, timeout: int = 120) -> None:
         """
         start_time = time.time()
         while time.time() - start_time < timeout:
-            if self.get_application_has_converged():
-                return
-            else:
-                logger.debug('Waiting for Vespa application to converge')
-                time.sleep(1)
+            try:
+                if self.get_application_has_converged():
+                    return
+                else:
+                    logger.debug('Waiting for Vespa application to converge')
+                    time.sleep(1)
+            except (httpx.TimeoutException, httpcore.TimeoutException):
+                pass
```

Comment:
* Why catching two errors?
* Swallowing error. Need to log at least. Log the error